### ISSUE TITLE
Problem with yt

### ISSUE THREAD

**Issue by mariakesa** [trident-project/trident#168]
Hello:-)

Thank you for an awesome package!

I have a problem with installation. I did an import trident and pulled the data files, but then I get the error message:

> ---------------------------------------------------------------------------
> ModuleNotFoundError                       Traceback (most recent call last)
> <ipython-input-20-e4535c710fbd> in <module>
>       1 import yt
> ----> 2 import trident
>       3 fn = 'enzo_cosmology_plus/RD0009/RD0009'
> 
> ~/anaconda3/lib/python3.7/site-packages/trident/__init__.py in <module>
>      39     atomic_mass
>      40 
> ---> 41 from trident.instrument import \
>      42     Instrument
>      43 
> 
> ~/anaconda3/lib/python3.7/site-packages/trident/instrument.py in <module>
>      15     YTQuantity
>      16 
> ---> 17 from trident.absorption_spectrum.absorption_spectrum import \
>      18     _bin_space_units
>      19 
> 
> ~/anaconda3/lib/python3.7/site-packages/trident/absorption_spectrum/absorption_spectrum.py in <module>
>      24 from yt.data_objects.static_output import \
>      25     Dataset
> ---> 26 from yt.convenience import load
>      27 from yt.funcs import get_pbar, mylog
>      28 from yt.units.yt_array import YTArray, YTQuantity
> 
> ModuleNotFoundError: No module named 'yt.convenience'

Any help would be appreciated!

**Comment by rjtanner** [comment]
Hi Maria,

I ran into this same problem two weeks ago. There were some changes from
yt3 to yt4 that caused this problem. There are two ways to fix this:

1. Download the development version of Trident where this has been fixed:
https://trident.readthedocs.io/en/latest/installation.html#step-2-install-trident

or

2. Go into the Trident files and change every instance of "yt.convenience"
to "yt.loaders". The error output will tell you which files need to be
changed. (i.e.
~/anaconda3/lib/python3.7/site-packages/trident/absorption_spectrum/absorption_spectrum.py)

Hope that helps.

Ryan Tanner
